client/src/components/MemberDetails/MemberDetails.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import GenericDialog from '../Form/GenericDialog/GenericDialog';
+import AnyUserNameWrapper from '../NameWrappers/AnyUserName/AnyUserNameWrapper'
+import { UIcontext } from '../contexts/UIcontext/UIcontext';
+import { ReactComponent as CopyIcon } from '../icons/copy.svg';
+import './MemberDetails.scss';
+
+ interface MemberDetailsProps {
+    closeDialog: () => void,
+    selectedMemberData: RegisteredUserType
+};
+
+const MemberDetails: React.FC<MemberDetailsProps> = ({ closeDialog, selectedMemberData: member }) => {
+    const [dataCopy, setDataCopy] = useState(false);
+    const { modalMembersDictionary, generalTermsDictionary, personalPagePlaceholders } = useContext(UIcontext).dictionary;
+    const { COPY, COPYTOCLIPBOARD } = modalMembersDictionary;
+    const { memberDetails } = useContext(UIcontext).dictionary;
+    const { SPIRITUALNAME, DATEOFBIRTH, GENDER, LEVELOFSTUDY, EMAIL, MOBILE, ADDRESS, EMNAME, EMMOBILE, EMEMAIL } = memberDetails;
+    const { modalButtons } = useContext(UIcontext).dictionary;
```

Comment:
lines 18 & 20 could be written in 1 line with 16.
I think you know that.
Is it more understandable the way you composed it?

Reply:
Copy&Paste original code. ;o)